ibc-clients/cw-context/src/types/error.rs

```diff
@@ -0,0 +1,36 @@
+use cosmwasm_std::StdError;
+use derive_more::{Display, From};
+use ibc_core::client::types::error::ClientError;
+use ibc_core::commitment_types::error::CommitmentError;
+use ibc_core::handler::types::error::ContextError;
+use ibc_core::host::types::error::IdentifierError;
+use ibc_core::host::types::path::PathError;
+use prost::DecodeError;
+
+#[derive(From, Display, Debug)]
+pub enum ContractError {
+    #[display(fmt = "CosmWasm standard error: {_0}")]
```

Comment:
interesting ! I didn't know `derive_more` has `Display` too.

we were using `displaydoc` in other sub-crates.